sqldelight-core/src/test/table-interface-fixtures/requires-adapter/Person.kt

```diff
@@ -21,12 +21,16 @@ interface Person {
 
   fun shhh_its_secret(): @Redacted String
 
-  data class Adapter(internal val friendsAdapter: ColumnAdapter<List<Person>, ByteArray>)
-
-  data class Impl(override val _id: Long, override val name: String,
-      override val last_name: String?, override val is_cool: Boolean,
-      override val friends: List<Person>?,
-      override val shhh_its_secret: @Redacted String) : PersonKt
+  class Adapter(internal val friendsAdapter: ColumnAdapter<List<Person>, ByteArray>)
+
+  data class Impl(
+    override val _id: Long,
```

Comment:
@Egorand 

minor nit but my editor by default does 4 spaces for the continuation indent - not sure if we want kotlinpoet to do the same

Reply:
Yes. File a bug.